src/components/IngredientCard/IngredientCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Themes from 'assets/json/themes.json';
+
+import './IngredientCard.scss';
+
+const IngredientCard = ({
+  name,
+  image,
+  theme,
+  children,
+}) => (
+  <div className={`ingredient-card ${Themes[theme] || Themes.default}`}>
```

Comment:
Use of `Theme[theme]` here is unnecessary! You're already passing `Theme.teal etc` to this comp

Reply:
This needs to be there. The backend will provide the theme to the frontend. And that will be the key. Currently I am using Themes.teal on `Dashboard` till the backend is done. `IngredientCard` would still have this code.

Reply:
My point being, over here you will **`always`** get a value cause you're passing it from the parent. Use of default should be moved to the parent.

Reply:
But someone creating a component might not pass that prop. It is not a required prop so we need a default color on the component right?

Reply:
That is a weak argument to make something required. `theme` should not be required to render the component. By providing default in the component the intension is to make is optional.

Reply:
Ideally the them should always be provided. Even if its default. So if we leave this check here, we should still let a dev know that we did not receive a theme here and therefore used default